rust/src/variant_utils.rs

```diff
@@ -40,6 +41,11 @@ pub(crate) fn new_variant_array(ty: &glib::VariantTy, children: &[glib::Variant]
     }
 }
 
+pub(crate) fn new_variant_strv(strv: &[impl AsRef<str>]) -> glib::Variant {
```

Comment:
Ah heh I had a pending PR to add a variant of this to the ostree-rs-ext version, I will push it.

Reply:
Ahh, missed this one in the respin. Though looks like the PRs aren't completely in yet anyway. Let's clean that up in a follow-up?